Application/EdFi.Ods.Features/OpenApiMetadata/Factories/OpenApiMetadataPathsFactory.cs

```diff
@@ -253,20 +253,37 @@ private Operation CreatePutByIdOperation(OpenApiMetadataPathsResource openApiMet
                     OpenApiMetadataDocumentHelper.GetResourcePluralName(openApiMetadataResource.Resource)
                         .ToCamelCase()
                 },
-                summary = "Updates or creates a resource based on the resource identifier.",
-                description =
-                    "The PUT operation is used to update or create a resource by identifier. If the resource doesn't exist, the resource will be created using that identifier. Additionally, natural key values cannot be changed using this operation, and will not be modified in the database.  If the resource \"id\" is provided in the JSON body, it will be ignored as well.",
+                summary = "Updates a resource based on the resource identifier.",
+                description = GetDescription(openApiMetadataResource),
                 operationId = $"put{openApiMetadataResource.Name}",
                 deprecated = openApiMetadataResource.IsDeprecated,
                 consumes = new[]
                 {
                     _contentTypeStrategy.GetOperationContentType(openApiMetadataResource, ContentTypeUsage.Writable)
                 },
                 parameters = CreatePutParameters(openApiMetadataResource),
-                responses = OpenApiMetadataDocumentHelper.GetWriteOperationResponses(HttpMethod.Put)
+                responses = OpenApiMetadataDocumentHelper.GetWriteOperationResponses(HttpMethod.Put),
+                isUpdatable = CheckIsUpdatable(openApiMetadataResource)
```

Comment:
I found the name on this method misleading here.

On first read, it looked like you were simply checking the `IsUpdatable` property - effectively making it a function that should have been reused in the `GetDescription` method below as essentially a shorthand for the code of `openApiMetadataResource.Resource.Entity.Identifier.IsUpdatable`.

However, as I started writing comments I realized that the `CheckIsUpdatable` isn't so much about checking whether it's updatable, but it's actually preparing the flag for assignment to the Open API model as either `true` or `null` because it represents custom metadata and the JSON serializer will drop the property from the serialized output if it's `null`. That is not what the method name implies at all with the chosen verb "Check". Choosing good names is very important for making code maintainable.

Since you are actually preparing a specific model value for custom metadata in this method, please rename it to `GetIsUpdatableCustomMetadataValue`.

Then, shorten the implementation to this:
```
private bool? GetIsUpdatableCustomMetadataValue(OpenApiMetadataPathsResource openApiMetadataResource)
{
    return openApiMetadataResource.Resource.Entity.Identifier.IsUpdatable ? (bool?) true : null;
}
```

Reply:
done